src/main/java/util/Engine.java

```diff
@@ -6,73 +6,123 @@
 
 import static org.lwjgl.glfw.GLFW.glfwInit;
 
+/**
+ * The Engine class initializes GLFW and the game loop. <br\>
+ * Singleton.
+ */
 public final class Engine {
-
     /**
-     * The Engine class initializes GLFW and the game loop.
+     * The static {@code Engine} instance.
      */
-
     private static final Engine instance = new Engine();
-
+    
+    /**
+     * The exact time the engine started in milliseconds.
+     */
     private final long startMillis;
+    
+    /**
+     * No clue
+     */
```

Comment:
This is a reference to the Window class, which contains data like the glfw window, width, height, delta time and the main loop.

Reply:
oh ye, forgot i did that. was making a joke to prove my documenting skills xD
will fix that